molecule/default/prepare.yml

```diff
@@ -2,19 +2,19 @@
 ---
 - name: Prepare
   hosts: all
-  gather_facts: False
+  gather_facts: false
```

Comment:
The boolean values are intentionally written in uppercase, otherwise ansible-later (the used linting tool) will complain about it.

Reply:
Ahhh, ok. I'm too used to ansible-lint who's pretty annoying (by default) about FQDN usage and things like these boolean values. I'll get them flipped here in just a bit though!